src/Benchmarks/link-a-thon.api-template.json

```diff
@@ -13,29 +13,30 @@
     "Path": "/weatherforecast",
     "BuildArchives": [
       "https://aspnetbenchmarksbuilds.file.core.windows.net/singleexe/latest/files.zip?sv=2018-03-28&ss=f&srt=sco&sp=rl&se=2029-09-14T06:49:35Z&st=2019-09-13T22:49:35Z&spr=https&sig=rqCCHYVqkDJB2Z6mEPz5qeHxJJYEWmfJdI1XMtgv6zc%3D;../coreclrbin" 
-    ]
+    ],
+    "SelfContained": true
   },
   "LinkAThonBaseline": {
     "BuildArguments": [ ]
   },
   "LinkAThonTrimmedAndR2R": {
-    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true" ],
-    "SelfContained": true
+    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true" ]
   },
   "LinkAThonTrimmedAndR2RSingleFile": {
-    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true"],
-    "SelfContained": true
+    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true"]
   },
   "LinkAThonTrimmedAndR2RSingleFileWithTrimList": {
-    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true", "/p:UsePublishFilterList=true"],
-    "SelfContained": true
+    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true", "/p:UsePublishFilterList=true"]
   },
   "LinkAThonTrimmedAndR2RSingleFileNoMvc": {
-    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true", "/p:UsePublishFilterList=true", "/p:DefineConstants=NO_MVC"],
-    "SelfContained": true
+    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true", "/p:UsePublishFilterList=true", "/p:DefineConstants=NO_MVC"]
   },
   "LinkAThonTrimmedAndR2RSingleFileCustomHost": {
-    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true", "/p:UsePublishFilterList=true", "/p:DefineConstants=CUSTOM_BUILDER"],
-    "SelfContained": true
+    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true", "/p:UsePublishFilterList=true", "/p:DefineConstants=CUSTOM_BUILDER"]
+  },
+
+  "LinkAThonPlatformBaseline": {
+    "Project": "src/PlatformJson/PlatformJson",
```

Comment:
Have you tried it? It's missing the extension. Though it might still work.

Reply:
Also is that the one on the aspnet core repo? I don't remember you saying it locally

Reply:
it's in my repo. The problem was that it was missing `Source: { ... }` around it. And the file extension :)